contracts/GroupNFT.sol

```diff
@@ -125,10 +125,6 @@ contract GroupNFT is IGroupNFT, ERC721Upgradeable, AccessManagedUpgradeable, UUP
         return interfaceId == type(IGroupNFT).interfaceId || super.supportsInterface(interfaceId);
     }
 
-    ////////////////////////////////////////////////////////////////////////////
```

Comment:
Are there any other banners of "Upgrades related" in other licensing contracts to remove?

Reply:
removed all banners touched in the PR. I would prefer one PR for one specific work. clean up in other for files would be addressed in another PR.